src/index.html

```diff
@@ -1,25 +1,51 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>To-Do List</title>
-  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
-</head>
-<body>
-  <div class="main-container">
-    <header>
-      <h3>To-Do List</h3>
-      <i class="fa-solid fa-arrows-rotate"></i>
-    </header>
-    <form action="">
-      <input type="text" name="list" id="list" placeholder="Add to your list...">
-      <i class="fa-solid fa-arrow-turn-down fa-rotate-270 fa-sm"></i>
-    </form>
-    <ul class="list-container" id="tasks"></ul>
-    <button class="clear-btn" type="button">Clear all completed</button>
-  </div>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>To-Do List</title>
+  </head>
+  <body>
+    <div class="tasklist">
+      <nav>
+        <header>
+          <p>To-Do List</p>
+        </header>
+      </nav>
+      <div class="container">
+        <div class="newTask">
+          <div class="newTaskInput">
+            <input
+              type="text"
+              id="newTaskID"
+              placeholder="Add new task here..."
+            />
+          </div>
+          <div class="newTaskBtn">
+            <button class="AddTaskBtn">Add Task</button>
+            <button class="EditTaskBtn" >Edit To Do</button>
+            <button class="CancelTaskBtn">Cancel</button>
+          </div>
+        </div>
 
-  <script src="index.js"></script>
-</body>
-</html>
+        <div class="taskList">
+          <!-- <div class="task" data-createdat="15/07/2023, 6:59:55 PM">
+            <div class="taskDetails">
+              <input type="checkbox" class="task-check" />
+              <label class="taskTitle">Hello world</label>
+            </div>
+
+            <div class="taskIcon">
+              <ion-icon class="taskIconEdit" name="create-outline"></ion-icon>
+              <ion-icon class="taskIconDelete" name="trash-outline"></ion-icon>
+            </div>
+          </div> -->
```

Comment:
- [ ] If you think this piece of code is not relevant to the milestone, please make sure you remove it or uncomment it as it makes your code dirty and unprofessional, @Obote